packages/compass-saved-aggregations-queries/src/components/aggregations-queries-list.tsx

```diff
@@ -139,17 +155,7 @@ const AggregationsQueriesList = ({
         classNames={{ row: rowStyles }}
       ></VirtualGrid>
       <OpenItemModal></OpenItemModal>
-      {deletingItem && (
-        <DeleteItemModal
-          isOpen={true}
-          itemType={deletingItem.type}
-          onClose={() => setDeletingItem(undefined)}
-          onDelete={() => {
-            onDeleteItem(deletingItem.id);
-            setDeletingItem(undefined);
```

Comment:
Refactoring this aligns the implementation with [avoiding dispatching many actions sequentially](https://redux.js.org/style-guide/style-guide#avoid-dispatching-many-actions-sequentially) recommendation